mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -2,28 +2,30 @@
   "tables": [
     {
       "tableName": "job_executions",
-      "fromModuleVersion": "mod-source-record-manager-3.1.0",
+      "fromModuleVersion": "mod-source-record-manager-3.4.0",
       "withMetadata": false,
       "index": [
         {
           "fieldName": "id",
-          "tOps": "ADD"
+          "tOps": "DELETE"
```

Comment:
We should delete indexes for job_executions table before droping. There is _rmb_internal_analyze_ table which will be main cause of failure tests, because after schema updating the table data is used for analyzing. In my case EXPLAIN ANALYZE job_executions was my headache.

Reply:
Thanks for this additional info